catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/ModalRegistry.view.js

```diff
@@ -233,15 +234,24 @@ function (ich,Marionette,Backbone,ConfigurationEdit,wreqr,_,$,Utils,Service,moda
         },
         showError: function(msg) {
             var view = this;
-            var $group = view.$el.find('.registryName>.control-group');
-
-            $group.find('.error-text').text(msg).show();
+            var $group = view.$el.find('.registryUrl');
+            var $errorTextExists = $group.find('.error-text');
+            if ($errorTextExists.length===0) {
+                var $newdiv1 = $("<p id='error-text' class='error-text help-block col-md-12'></p>");
```

Comment:
Why can't this just be in the handlebars file and hidden by default? You could then just show/hide it here